libs/blocks/youtube/youtube.js

```diff
@@ -14,12 +14,18 @@ export default function init(a) {
         webkitallowfullscreen mozallowfullscreen allowfullscreen
         allow="encrypted-media; accelerometer; gyroscope; picture-in-picture"
         scrolling="no"
+        id="player-${id}"
         title="${title}">
       </iframe>
     </div>`;
     a.insertAdjacentHTML('afterend', embedHTML);
     a.remove();
+    document.addEventListener('readystatechange', () => {
```

Comment:
Is it possible that the event is added after the `readyState` is already `complete`? If so, it should first be checked if the `document.readyState` is already `complete`. If so, call the satellite method. If not, add the event listener.

Reply:
I all the scenarios we have not seen this case when tested at our end. As this is the loading state of the document, it is unlikely this will happen.